docs/source/resources/transition_to_ft_v1.0.ipynb

```diff
@@ -30,17 +30,34 @@
     "- Both the `Entity` and `Variable` classes have been removed from Featuretools\n",
     "- Several key Featuretools methods have been moved or updated\n",
     "\n",
-    "#### Comparison between legacy and new typing systems\n",
+    "#### Comparison between legacy typing system and Woodwork typing systems\n",
     "| Featuretools < 1.0 | Featuretools 1.0 | Description |\n",
     "| ---- | ---- | ---- |\n",
     "| `Entity` | Woodwork DataFrame | stores typing information for all columns |\n",
     "| `Variable` | `Column Schema` | stores typing information for a single column |\n",
-    "| `Variable` subclass | `LogicalType` and `semantic_tags` | elements used to define a column type |"
+    "| `Variable` subclass | `LogicalType` and `semantic_tags` | elements used to define a column type |\n",
+    "\n",
+    "#### Summary of significant method changes\n",
+    "\n",
+    "The table below outlines the most significant changes that have occurred. In some cases, the method arguments have also changes, and those changes are outlined in more detail throughout this document.\n",
+    "\n",
+    "| Older Versions | Featuretools 1.0 |\n",
+    "| ---- | ---- |\n",
+    "| EntitySet.entity_from_dataframe | [EntitySet.add_dataframe](#Adding-dataframes-to-an-EntitySet) |\n",
+    "| EntitySet.normalize_entity | [EntitySet.normalize_dataframe](#Normalizing-Entities/DataFrames) |\n",
+    "| EntitySet.update_data | [EntitySet.replace_dataframe](#Updating-data-for-a-dataframe-in-an-EntitySet) |\n",
+    "| Entity.variable_types | [es['dataframe_name'].ww](#Accessing-column-typing-information) |\n",
+    "| es['entity_id']['variable_name'] | [es['dataframe_name'].ww.columns['column_name']](#Accessing-column-typing-information) |\n",
+    "| Entity.convert_variable_type | [es['dataframe_name'].ww.set_types](#Type-inference-and-updating-column-types) |\n",
+    "| Entity.add_interesting_values | [es.add_interesting_values(dataframe_name='df_name', ...)](#Adding-interesting-values) |\n",
+    "| Entity.set_secondary_time_index | [es.set_secondary_time_index(dataframe_name='df_name', ...)](#Setting-a-secondary-time-index) |\n",
+    "| Feature(es['entity_id']['variable_name']) | [Feature(es['dataframe_name'].ww['column_name'])](#Defining-features) |\n",
+    "| dfs(target_entity='entity_id', ...) | [dfs(target_dataframe_name='dataframe_name', ...)](#Changes-to-Deep-Feature-Synthesis-and-Calculate-Feature-Matrix) |"
```

Comment:
Looks great--I like how it links to the relevant parts of the guide while also providing a quick mapping between the old and new api 
![image](https://user-images.githubusercontent.com/64278226/131559298-40db8f1c-5f0b-4efe-83ed-ffcc4af52032.png)

A couple of things I'm seeing from the readthedocs build:
- seems like there's an extra `_` after `normalize_dataframe` - but I don't see it in the raw cell
- `ww.set_types` has an extra space between the p and e? "typ es"
- `es.add_interesting_values` doesn't seem to be formatting the link correctly
- `dataframe_name` in the dfs call also has a space that I don't see in the raw cell: "’data frame_name’, …)"

Reply:
would it be easier to link to the api refrence? Would mean that you weren't linking to descriptions about the changes, but might still be useful

Reply:
In some cases, perhaps, but that's probably not quite as clean for the typing related changes that go through the WW accessor...

Reply:
Definitely not the end of the world if the links don't make it. We can just let readers know that further descriptions are later on in the guide

Reply:
I'm not seeing anything right away that would cause this weird formatting. I'm thinking it's probably not worth spending a whole of time trying to fix at this point, so I will remove the links and switch the table to use code formatting instead.

Reply:
I ended up removing both the links and the code formatting inside of the table as read the docs seems to have problems with rendering both correctly.